var/spack/repos/builtin/packages/cbtf-argonavis-gui/package.py

```diff
@@ -51,50 +51,44 @@ class CbtfArgonavisGui(QMakePackage):
     url = "https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git"
 
     version('1.3.0.0', branch='1.3.0.0',
-            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
+            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git', preferred=True)
```

Comment:
This change shouldn't be necessary, Spack should already be choosing 1.3.0.0 by default.

Reply:
I think it was picking 1.3.0.0 as you say without the preferred, but spack was not following the same path as if the preferred statement clause is present.  The build was getting concretization errors without the preferred clause.